6.6/scripts/oom_solr.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# This is just a wrapper to make it actually work in the start script - because
+# the whitespaces of belows call cause havoc otherwise:
+#. /opt/solr/bin/oom_solr.sh ${SOLR_PORT:-8983} /opt/solr/server/logs
```

Comment:
I'm not entirely sure what you mean there?
I don't think you want the leading period.

Reply:
Originally I just wanted to trigger the oom_solr.sh shipped by solr https://github.com/apache/lucene-solr/blob/master/solr/bin/oom_solr.sh
But it seems to be reasonable to use our own implementation by now.